python/cudf/cudf/testing/io.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+import copy
+import json
+import logging
+import os
+import random
+import sys
+
+logging.basicConfig(
+    format="%(asctime)s %(levelname)-8s %(message)s",
+    level=logging.INFO,
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+
+class IOFuzz(object):
+    def __init__(
+        self,
+        dirs=None,
+        max_rows=100_000,
+        max_columns=1000,
+        max_string_length=None,
+    ):
+        self._inputs = []
+        self._max_rows = max_rows
+        self._max_columns = max_columns
+        self._max_string_length = max_string_length
+
+        for i, path in enumerate(dirs):
```

Comment:
This will fail if `dirs` is None, which is the default.

Reply:
Good catch @cwharris , This was being filtered out in fuzzer so didn't error, added same conditional check here aswell.